pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -1113,7 +1114,7 @@ private ConsumerImpl<T> createInternalConsumer(ConsumerConfigurationData<T> conf
         return ConsumerImpl.newConsumerImpl(client, partitionName,
                 configurationData, client.externalExecutorProvider(),
                 partitionIndex, true, listener != null, subFuture,
-                startMessageId, schema, interceptors,
+                startMessageId, schema, null,
```

Comment:
we can't remove interceptors from internal consumer directly this will make other interceptor methods not work